### ISSUE TITLE
a-scene memory leak on chrome

### ISSUE THREAD

**Issue by Dirk-27** [aframevr/aframe#4899]
**Description:**

- A-Frame Version: 1.2.0 and master
- Platform / Device: Chrome Desktop (Version 92.0.4515.131 (64-Bit))
- Reproducible Code Snippet or URL: https://jsfiddle.net/neomatt/4jw6xyhe/8/

Creating an a-scene allocates memory that is not released. This behaviour can be observed in the chrome browser (not in firefox). The amount of memory used depends on the size of the browser window. The memory used seems to be associated with the browser window even after removing the a-scene tag. So if you increase the size of the window after running the jsfiddle script, you can observe a higher memory usage in Chrome.

**Comment by Benjy763** [comment]
Hi, I also have a big problem with chrome about memory. With Firefox it is fine, but with chrome the memory increases very quickly, it is more than 5 times the memory allocated on FF.
There is a thread here https://github.com/aframevr/aframe/issues/3137 But this is a problem relative to chrome.

Hope you'll find a solution, thanks again !

**Comment by dmarcos** [comment]
There's a proposed solution here https://github.com/aframevr/aframe/issues/3137#issuecomment-990201997

**Comment by Dirk-27** [comment]
Right. Only for clarification: This issue here (#4899) exists independent from models in the scene.

**Comment by dmarcos** [comment]
Any ideas why the behavior is different between Firefox and Chrome?

**Comment by Dirk-27** [comment]
No. I think it is webgl related. Interesting is: 'The amount of memory used depends on the size of the browser window.'
So the bug has perhaps to do with the use of some graphic buffers?

I found a chrome issue that fits to this issue: https://bugs.chromium.org/p/chromium/issues/detail?id=811220

**Comment by dmarcos** [comment]
Thanks. I leave this opened as a reference
